tsconfig.json

```diff
@@ -1,19 +1,28 @@
 {
     "compilerOptions": {
+        "target": "ESNext",
         "module": "ESNext",
         "moduleResolution": "Bundler",
         "types": ["node", "vitest/globals"],
-        "declaration": true,
```

Comment:
@acaldas what was the purpose for setting these options to only emit declarations? I ask because with these configs all of the stories in storybook have [this issue happening](https://github.com/microsoft/TypeScript/issues/42873). I can probably find a workaround, but the easiest solution is to just turn these off. Please let me know if this will cause other problems that I'm not seeing here

Reply:
I guess we don't need id since we use `vite-plugin-dts` to bundle the types. I don't remember why it is configured this way.